redoctober.go

```diff
@@ -134,9 +134,9 @@ func NewServer(process chan<- userRequest, staticPath, addr, caPath string, cert
 			log.Fatal(err)
 		}
 		if len(listenFDs) != 1 {
-			log.Fatal("Unexpected number of socket activation FDs!")
+			log.Fatalf("Unexpected number of socket activation FDs! (%v)", len(listenFDs))
```

Comment:
Might as well call it a `%d`, it's a bit clearer to specify the type if you know it.